public/app/plugins/panel/geomap/utils/checkFeatureMatchesStyleRule.ts

```diff
@@ -0,0 +1,30 @@
+import { Feature } from 'ol';
+import { Geometry } from 'ol/geom';
+
+export interface GeoJSONMapperRule {
+  property: string;
+  operation: string;
```

Comment:
can operation use an enum rather than string that is defined thoughout?

Then the rest of the 'gt', 'gte', ... can be constants rather than inline strings